lib/components/ConnectModal/index.tsx

```diff
@@ -134,13 +148,25 @@ export const ConnectModal = (props: ConnectModalProps) => {
                       wallet: showCoinbaseWallet ? "injected" : "coinbase",
                     })
                   }
+                  onKeyDown={(e) =>
+                    e.code === "Enter" &&
+                    handleConnect({
+                      wallet: showCoinbaseWallet ? "injected" : "coinbase",
+                    })
+                  }
+                  tabIndex={5}
                 >
                   <CoinbaseWalletIcon />
                   <Text t="button">Coinbase</Text>
                 </span>
                 <span
```

Comment:
this is why focusTrap doesn't work-- only anchor tags, buttons and inputs are focusable. The focustrap hook grabs all buttons and injects the listeners for us.

So you can rip out all the keydown and tabindex stuff and just make sure every focusable element is a button

Reply:
Ok! Yea that makes much more sense than adding tab index. TY!

Reply:
updated to use proper html tags and now using css to show/hide elements so that the focus trap works properly when in error step and bc less code. Fun fact: MM doesn't support tabbing.